dags/fxa_amplitude_export.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+
+from airflow import models
+from utils.gcp import bigquery_etl_query
+from airflow.operators.subdag_operator import SubDagOperator
+from utils.amplitude import export_to_amplitude
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'start_date': datetime.datetime(2020, 5, 1),
+    'email': ['telemetry-alerts@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'depends_on_past': True,
```

Comment:
I don't think we should set this at the dag level, but instead just mark `fxa_amplitude_export_v1` as depends_on_past